### PR TITLE
Blog post: Enhance OpenAI with Binary Quantization

### PR DESCRIPTION
**Pull request by mjang**

### CODE REVIEW

File: qdrant-landing/content/blog/binary-quantization-openai.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+- Qdrant's Binary Quantization, and how it can improve the performance of OpenAI embeddings
+- Results of an experiment that highlights improvements in search efficiency and accuracy
+- Implications of these findings for real-world applications
+- Best practices for leveraging Binary Quantization to enhance OpenAI embeddings
+
+## New OpenAI Embeddings: Performance and Changes
+
+As the technology of embedding models has advanced, demand has grown. Users are looking more for powerful and efficient text-embedding models. OpenAI's Ada-003 embeddings offer state-of-the-art performance on a wide range of NLP tasks, including those noted in [MTEB](https://huggingface.co/spaces/mteb/leaderboard) and [MIRACL](https://openai.com/blog/new-embedding-models-and-api-updates). 
+
+### Multi-lingual Support
```
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:37]:
```suggestion
### Multilingual Support
```

File: qdrant-landing/content/blog/binary-quantization-openai.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+We use 100K random samples from the [OpenAI 1M](https://huggingface.co/datasets/KShivendu/dbpedia-entities-openai-1M) dataset. We select 100 records at random from the dataset. We then use the embeddings of the queries to search for the nearest neighbors in the dataset. 
+
+#### Parameters: Oversampling, Rescoring, and Search Limits
+
+For each record, we run a parameter sweep over the number of oversampling, rescoring, and search limits. We can then understand the impact of these parameters on search accuracy and efficiency. Our experiment was designed to assess the impact of Binary Quantization under various conditions, based on the following parameters: 
+
+- Oversampling
+- Rescoring
+- Search limits
+
```
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:74]:
```suggestion
```

File: qdrant-landing/content/blog/binary-quantization-openai.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+- **Search Limits**: We specify the number of results from the search process. We experimented with various search limits to measure their impact the accuracy and efficiency. We explored the trade-offs between search depth and performance. The results provide insight for applications with different precision and speed requirements.
+
+Through this detailed setup, our experiment sought to shed light on the nuanced interplay between Binary Quantization and the high-quality embeddings produced by OpenAI's models. By meticulously adjusting and observing the outcomes under different conditions, we aimed to uncover actionable insights that could empower users to harness the full potential of Qdrant in combination with OpenAI's embeddings, regardless of their specific application needs.
+
+### Results: Binary Quantization's Impact on OpenAI Embeddings
+
+To analyze the impact of rescoring (`True` or `False), we compared results across different model configurations and search limits. Rescoring sets up a more precise search, based on results from an initial query.
```
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:81]:
```suggestion
To analyze the impact of rescoring (`True` or `False`), we compared results across different model configurations and search limits. Rescoring sets up a more precise search, based on results from an initial query.
```

**Review by sabrinaaquino — APPROVED** [review]
I've added a few modifications. Lgtm but would like a final approval.

File: qdrant-landing/content/blog/binary-quantization-openai.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+In this post, we discuss:
+
+- The significance of OpenAI embeddings and real-world challenges. 
+- Qdrant's Binary Quantization, and how it can improve the performance of OpenAI embeddings
+- Results of an experiment that highlights improvements in search efficiency and accuracy
+- Implications of these findings for real-world applications
+- Best practices for leveraging Binary Quantization to enhance OpenAI embeddings
+
+You can also try out these techniques with the following Jupyter notebook: [OpenAI Embeddings with Qdrant's Binary Quantization](https://github.com/qdrant/examples/blob/openai-3/binary-quantization-openai/analysis/01_analysis.ipynb)
```
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:35]: I'm not sure if we should refer to the Jupyter notebook or any other instructions _until_ @NirantK merges the `openai-3` branch into the `examples` repository.

(Maybe we should hold off merging until after Nirant merges his work into the `examples` repo.)

If we do, we'll have to change the link after the merge. And I think we should start with a different page:

```suggestion
You can also try out these techniques as described in [Binary Quantization OpenAI](https://github.com/qdrant/examples/blob/openai-3/binary-quantization-openai/README.md), which includes Jupyter notebooks.
```

After Nirant merges the change, the tech would change to:

```suggestion
You can also try out these techniques as described in [Binary Quantization OpenAI](https://github.com/qdrant/examples/blob/master/binary-quantization-openai/README.md), which includes Jupyter notebooks.
```

File: qdrant-landing/content/blog/binary-quantization-openai.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+## New OpenAI Embeddings: Performance and Changes
+
+As the technology of embedding models has advanced, demand has grown. Users are looking more for powerful and efficient text-embedding models. OpenAI's Ada-003 embeddings offer state-of-the-art performance on a wide range of NLP tasks, including those noted in [MTEB](https://huggingface.co/spaces/mteb/leaderboard) and [MIRACL](https://openai.com/blog/new-embedding-models-and-api-updates). 
+
+ A notable feature of these models is their multi-lingual support, enabling encoding in over 100 languages, which addresses the needs of applications with diverse language requirements. Impressively, the transition from text-embedding-ada-002 to text-embedding-3-large has observed a significant jump in performance scores (from 31.4% to 54.9% on MIRACL), reflecting substantial advancements.
```
Comment (mjang) [inline qdrant-landing/content/blog/binary-quantization-openai.md:41]:
```suggestion
These models include multilingual support in over 100 languages. The transition from text-embedding-ada-002 to text-embedding-3-large has led to a significant jump in performance scores (from 31.4% to 54.9% on MIRACL).
```

**Review by mjang — COMMENTED** [review]
Open question: should we include a link to the Jupyter notebook examples at all before Nirant merges.

**Review by sabrinaaquino — APPROVED** [review]

**Comment by sabrinaaquino** [comment]
Let's not include the notebook as it's not complete yet. We can revisit this later.

**Pull request merged**